astroquery/tests/test_internet.py

```diff
@@ -1,14 +1,24 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from astropy.tests.helper import remote_data
 import requests
+import os
 
 from .. import version
 
+IS_CI = os.environ.get('TRAVIS', False) or os.environ.get('APPVEYOR', False)
+
 
 @remote_data
 def test_net_connection():
-    R = requests.post('http://httpbin.org/post',
-                      headers={'User-Agent':
-                               'astropy:astroquery.{0}'
-                               .format(version.version)})
-    R.raise_for_status()
+    if IS_CI is False:
```

Comment:
I suggest only to set a string on this status, and substitute it in below in one `R = requests.post(...)` statement rather than repeat everything twice.

Reply:
Okay, so I should do something like `IS_CI = os.environ.get('TRAVIS', False) or os.environ.get('APPVEYOR', False) = True`? Then use just one `R = requests.post(...)`? Surely I can do that...

Reply:
Done